LayoutTests/accessibility/duplicate-axrenderobject-crash.html

```diff
@@ -13,15 +14,15 @@
     </summary>
 </label>
 
-<p id="description"></p>
-
 <script>
-    description("Ensures that it's not possible to have two AXRenderObjects that point to the same renderer, if the initialization of an AXRenderObject results in another object with the same renderer being created before AXObjectCache has added that mapping to its hash.");
-    if (window.accessibilityController)
+    var output = "Ensures that it's not possible to have two AXRenderObjects that point to the same renderer, if the initialization of an AXRenderObject results in another object with the same renderer being created before AXObjectCache has added that mapping to its hash.";
+
+    if (window.accessibilityController) {
         accessibilityController.accessibleElementById("dummy");
+    }
```

Comment:
I think WebKit's style guide suggests not including curly-braces for single-statement `if`s.